.github/workflows/beta.yml

```diff
@@ -4,27 +4,50 @@ on:
   push:
     branches:
       - 'beta'
+  workflow_dispatch: 
 
+env:
+  IMAGE: todrob/searcharr
 jobs:
   docker:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
     steps:
-      -
-        name: Set up QEMU
-        uses: docker/setup-qemu-action@v2
-      -
-        name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v2
+      - 
+        uses: actions/checkout@v2
       -
         name: Login to DockerHub
         uses: docker/login-action@v2
         with:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
-      -
+      - if: startsWith(github.ref, 'refs/heads/master')
+        run: echo "TAG=latest" >> $GITHUB_ENV
+      - if: startsWith(github.ref, 'refs/tags')
+        run: echo "TAG=$(git describe --tags)" >> $GITHUB_ENV
+      - name: upgrade apt packages..
+        run: sudo apt-get update -y && sudo apt-get upgrade -y
```

Comment:
Why bother updating--are any apt packages even being used?

Reply:
no but its wroth it keep latest packages uptodate and i have also used ```apt-get autoremove -y```